spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaFileIndex.scala

```diff
@@ -130,6 +130,7 @@ private[sharing] abstract class RemoteDeltaCDFFileIndexBase(
     auxPartitionSchema: Map[String, DataType] = Map.empty)
     extends RemoteDeltaFileIndexBase(params) {
 
+  // TODO: linzhou, update this
```

Comment:
Consider adding more details on what needs to be updated.

Reply:
This need to be removed.